README.md

```diff
@@ -50,6 +50,7 @@ Docker-compose
    ```
    cp adx_develop/dev.env adx_develop/.env
    ```
+   You may need to change `ADR_HOSTNAME=adr.local` in `adx_develop/.env`
```

Comment:
that's the default value in `dev.env`. updated with:
https://github.com/fjelltopp/adx_develop/commit/a3b650356073660b3617c2c878e6981c6678f016

Reply:
🤔  oh hah, you're right